team/_posts/2022-10-15-david-castillo.md

```diff
@@ -13,3 +13,4 @@ image: /assets/images/team/DCastillo.jpg
 cv: 
 alum: false
 ---
+I'm a microbiologist interested in understanding how cells move and utilize energy. Throughout my career, I researched molecular nanomotors including the type IV pili, ATP synthase and the flagellar motor of different bacterial species. Despite extensive years of research, I find the flagellar motor to be one of the most intricate and complex prokaryotic structures. It is an excellent model for comprehending core fundamental concepts regarding biological evolution of complex systems at the nanoscale level.  
```

Comment:
In the last sentence, 
- Can you either "core" or "fundamental" (they both mean sort of the same thing)?
- Can you replace "biological" (isn't all evolution biological) with "the"?